### PR TITLE
Fix isempty with Polyhedra backend

### PR DESCRIPTION
**Pull request by mforets**
See #2532.

### CODE REVIEW

File: src/Sets/HPolyhedron.jl [hunk 1]
```diff
@@ -483,7 +483,7 @@ Otherwise, we set up the LP internally.
 function isempty(P::HPoly{N},
                  witness::Bool=false;
                  use_polyhedra_interface::Bool=false,
-                 solver=default_lp_solver(N),
+                 solver=nothing,
```
Comment (schillic) [inline src/Sets/HPolyhedron.jl:486]: I guess the docs need to be adapted now.

**Review by schillic — APPROVED** [review]
Can you merge/rebase with master? I expect that this fixes the build error.

File: src/Sets/HPolyhedron.jl (hunk 1 above)
Comment (mforets) [reply src/Sets/HPolyhedron.jl:487]: done. on the other hand, having `(optional, default: `nothing`)` is sort of meaningless so i added a comment in that line

**Comment by mforets** [comment]
the PR was approved and the build error in windows is unrelated ... merging !

**Pull request merged**